Resources/Prototypes/Voice/disease_emotes.yml

```
@@ -17,7 +27,7 @@
     components:
     - BorgChassis
   chatTriggers:
-    - coughs
+    - "chat-emote-trigger-cough-1" # WD EDIT
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)